src/psyclone/gocean1p0.py

```diff
@@ -1427,15 +1426,16 @@ def acc_args(self):
         # access.
         grid_fld = self.find_grid_access()
         grid_ptr = grid_fld.name + "%grid"
-        arg_list.extend([grid_fld.name, grid_fld.name+"%data"])
-
+        api_config = Config.get().api_conf("gocean1.0")
+        data_fmt = api_config.field_properties["go_grid_data"][0]
```

Comment:
Again, I think this should be a 'field property', not a grid property.

Reply:
Yes, left this for #676.

Reply:
Please add a TODO and reference that issue.